### PR TITLE
Update nagwrap to handle unresponsive checks

### PR DESCRIPTION
**Pull request by GrayTShirt**

### CODE REVIEW

File: nagwrap [hunk 1]
```diff
@@ -7,40 +7,59 @@ use Getopt::Long qw/:config bundling/;
...
 	unless $OPTIONS{prefix} and @ARGV;
 
-chomp(my $fqdn = qx(/bin/hostname -f));
+chomp(my $fqdn = qx(/bin/hostname));
 $OPTIONS{prefix} =~ s/^:/$fqdn:/;
 
-my $pid = open2(\*CHILD_OUT, \*CHILD_IN, @ARGV);
-defined $pid or die "open2 failed: $!\n";
+if (!-f $ARGV[0]) {
+	printf "STATE %i %s CRITICAL cannont find check plugin %s\n", time, $OPTIONS{prefix}, $ARGV[0];
```
Comment (jhunt) [inline nagwrap]: cannont?

File: nagwrap [hunk 2]
```diff
@@ -7,40 +7,59 @@ use Getopt::Long qw/:config bundling/;
...
 GetOptions \%OPTIONS, qw/
 	prefix|n=s
+	timeout|t=i
 /;
+
 die "USAGE: nagwrap -n :name:of:check -- /original/nagios/plugin [options]\n"
 	unless $OPTIONS{prefix} and @ARGV;
 
-chomp(my $fqdn = qx(/bin/hostname -f));
+chomp(my $fqdn = qx(/bin/hostname));
```
Comment (jhunt) [inline nagwrap]: Is dropping the `-f` pertinent to the timeout issue?

Comment (GrayTShirt) [inline nagwrap]: it's not a timeout issue.

It's a namespace issue with `hostname' and reverse DNS. The quick solution was to drop '-f' as the ptr record points to a reverse mapping of the server elevation in the DC.

File: nagwrap (hunk 1 above)
Comment (GrayTShirt) [inline nagwrap]: reponses :)

File: nagwrap (hunk 2 above)
Comment (jhunt) [inline nagwrap]: Can you pull that change from this PR and raise a separate issue with more context?

Comment (GrayTShirt) [inline nagwrap]: okay, I pulled the change

**Pull request merged**